fluentd/run.sh

```diff
@@ -19,7 +19,7 @@ done
 echo "============================="
 echo "Fluentd logs have been redirected to: $LOGGING_FILE_PATH"
 echo "If you want to print out the logs, use command:"
-echo "oc exec <pod_name> $HOME/utils/logs"
+echo "oc exec <pod_name> /usr/local/bin/logs"
```

Comment:
Does `oc exec <pod_name> -- logs` work?  If not, we need to find out why the `logs` command is not in the default PATH.